### DISCUSSION TITLE
How do I create a lora for Flux using Kohya?

### DISCUSSION THREAD

**Discussion by Deejay85** [bmaltais/kohya_ss#3001] [Q&A]
I've selected a base Flux model, but there is no option to train things for Flux. I've updated to the most recent version of Kohya, even doing a fresh install, but still don't have the option available to create a lora using Flux models.

**Comment by 5KilosOfCheese** [comment]
Did you select the "SD3-Flux.1" branch? It should be just in the LoRA menu.

**Reply by TanvirHafiz** [reply]
where is the branch or how can i go to the branch?

**Reply by ZfE0QQ6ds92W** [reply]
git clone --SD3-Flux.1 https://github.com/bmaltais/kohya_ss.git

**Reply by abellos** [reply]
> git clone --SD3-Flux.1 https://github.com/bmaltais/kohya_ss.git

i get this error

error: unknown option `SD3-Flux.1'

**Comment by Sylsatra** [comment]
git clone -b sd3-flux.1 https://github.com/bmaltais/kohya_ss.git

**Comment by LichAcademy** [comment]
I know it's been a while, but to anyone reading this in 2025:

You still need `--recursive` flag, so that it fetches actual `sd-scripts` (repo) from `kohya_ss` (user):

```
git clone --branch sd3-flux.1 --recursive https://github.com/bmaltais/kohya_ss.git kohya_sd35
```

(`kohya_sd35` at the end is optional; I like to rename my repo, so I know it is not the main `kohya_ss` branch)
